icom/ic7300.c

```diff
@@ -285,6 +300,176 @@ const struct rig_caps ic7300_caps = {
 
 };
 
+const struct rig_caps ic9700_caps = {
```

Comment:
Great to see initial IC-9700 here, but in my opinion these changes would already need a separate file `ic9700.c` with IC-9700-specific constants. You can make the ic7300 rit / other functions non-static to share them... but since 9700 is not a revision of 7300, I would _not_ put the rig definitions to the same file.